tetris.py

```diff
@@ -125,29 +132,63 @@ def new_block(self):
             self.current_block_shape = self.upcoming_block_shape
         self.current_block_center = (int(game_width / 2), -2)
         self.upcoming_block_shape = random.choice(shape_names)
+        self.rotate_counter = 0
 
     def get_current_block(self):
         (x, y) = self.current_block_center
         if self.current_block_shape == "I":
-            return [(x - 2, y), (x - 1, y), (x, y), (x + 1, y)]
+            I = [
+                [(x - 2, y), (x - 1, y), (x, y), (x + 1, y)],
+                [(x, y + 1), (x, y), (x, y - 1), (x, y - 2)],
```

Comment:
I would arrange the coordinates so that below each coordinate is a rotated version of that coordinate. For example, below `(x - 2, y)` is `(x, y - 2)`. That way it's easier to see what's going on.

```suggestion
                [(x - 2, y), (x - 1, y), (x, y), (x + 1, y)],
                [(x, y - 2), (x, y - 1), (x, y), (x, y + 1)],
```

Reply:
Man, as soon as I get used to a system *you* have suggested, you come up with another one 🤪